Stylesheets/components/cards.css

```diff
@@ -0,0 +1,140 @@
+@import url(/Stylesheets/main.css);
+body {
+  background-color: aquamarine;
+}
+.container {
+  margin-top: 1rem;
+  display: flex;
+  flex-wrap: wrap;
+  row-gap: 2rem;
+}
+.card {
+  height: 21.5rem;
+  width: 15rem;
+  background: var(--bg-white);
+  margin-left: 1rem;
+}
+.card2 {
+  height: 21.5rem;
+  width: 15rem;
+  background: var(--bg-white);
+  margin-left: 1rem;
+  box-shadow: 7px 12px grey;
+}
+.card-only {
+  height: 11.5rem;
+  width: 15rem;
+  background: var(--bg-white);
+  margin-left: 1rem;
+  box-shadow: 20, 20 blue;
+}
+
+.card_image {
+  background-color: rgb(25, 0, 255);
+  height: 10rem;
+  margin-bottom: 0.3rem;
+  background-size: cover;
+}
+
+.card_image2 {
+  height: 10rem;
+  width: 10rem;
+  background-size: cover;
+}
+
+.card-1 {
+  background-image: url("https://images.unsplash.com/photo-1542291026-7eec264c27ff?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MXx8c2hvZXxlbnwwfHwwfHw%3D&w=1000&q=80");
+}
+
+.badge1 {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  position: relative;
+  top: 0.5rem;
+  left: 0.2rem;
+  background-color: orange;
+  height: 2rem;
+  width: 6rem;
+  font-size: small;
+  text-align: center;
+}
+
+img {
+  position: relative;
+}
+
+.text {
+  position: relative;
```

Comment:
try to avoid repeating code. You can club these classes together for repeating code.

Reply:
okay. done :)